Source/Core/DolphinQt/MainWindow.cpp

```diff
@@ -163,6 +158,17 @@ void DMainWindow::StartGame(const QString filename)
 		}
 		m_render_widget->adjustSize();
 	}
+	else
+	{
+		m_render_widget->showMaximized();
```

Comment:
I don't think this is what we want. DolphinWX shows the widget non-maximized, shouldn't we do the same?